### DISCUSSION TITLE
这个MessageBox的按钮为啥是中文显示的？

### DISCUSSION THREAD

**Discussion by Farb** [HandyOrg/HandyControl#1436] [Q&A] [answered]
![image](https://github.com/HandyOrg/HandyControl/assets/7587900/2a5ccbd0-1c81-4a62-9613-62b17a501599)

如题，我已经在切换语言的地方，设置了线程和默认文化信息。
![image](https://github.com/HandyOrg/HandyControl/assets/7587900/2b6859d2-f74e-40b7-96e0-0ada95baf528)

**Comment by NaBian** [comment, marked as answer]
nuget 下载对应语言包

**Reply by linshuchang** [reply]
我的日历控件一半是中文一半是英文，怎么回事？
![PixPin_2024-11-15_14-34-46](https://github.com/user-attachments/assets/9fc3f286-0523-415e-bae7-f63be1df3109)

**Reply by linshuchang** [reply]
![PixPin_2024-11-15_18-08-13](https://github.com/user-attachments/assets/cfcb79ea-de4f-4e86-9668-debf03d037e5)
